src/management-system-v2/components/html-form-editor/elements/Input.tsx

```diff
@@ -41,6 +41,16 @@ export const ExportInput: UserComponent<InputProps> = ({
     />
   );
 
+  const input = (
+    <input
+      id={inputId}
+      className={variable ? `variable-${variable}` : undefined}
+      type={type}
+      defaultValue={value}
+      name={variable}
+    />
+  );
+
   return (
```

Comment:
This seems unintentional as the same input was already defined above
```suggestion
  return (
```